polyfactory/value_generators/constrained_numbers.py

```diff
@@ -1,8 +1,9 @@
 from __future__ import annotations
 
 from decimal import Decimal
+from math import ceil, floor, ulp
```

Comment:
It looks like I'm failing the Python 3.8 tests because I'm importing [`ulp`](https://docs.python.org/3/library/math.html#math.ulp), which was only added in Python 3.9. I'll probably have to implement an alternative way of computing the correct increment for a given floating-point number.